portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,27 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  ArrayList<String> messages = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+
+    // Create arrayList that will contain few hardcoded messages.
+    messages.add("Hello it is sooo nice to see you!");
+    messages.add("How are you doing?");
+    messages.add("What is your name?");
+
+    //Convert arraylist data into json
+    String json = convertToJsonUsingGson(messages);
+   
+    //Send JSON as response to the client
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
   }
+
+  private String convertToJsonUsingGson(ArrayList<String> messages) {
```

Comment:
```suggestion
  private String convertToJsonUsingGson(List<String> messages) {
```

Same here, make it List.

Reply:
Thank you, just fixed it.